core-bundle/contao/themes/flexible/styles/main.css

```
@@ -2243,6 +2249,7 @@ ul.sgallery li {
 	}
 }
 
+.tl_modulewizard td:first-child,
 .tl_modulewizard td:last-child {
```

```suggestion
.tl_modulewizard td:is(:first-child, :last-child) {
```
Just a nit, ignore if not needed :)